lib/ancestry/instance_methods.rb

```diff
@@ -76,7 +76,7 @@ def touch_ancestors_callback
         if self.ancestry_base_class.touch_ancestors
 
           # Touch each of the old *and* new ancestors
-          self.class.where(id: (ancestor_ids + ancestor_ids_was).uniq).each do |ancestor|
+          self.class.unscope(where: :type).where(id: (ancestor_ids + ancestor_ids_was).uniq).each do |ancestor|
```

Comment:
Would it make sense to use `ancestry_base_class` here instead?

Also, could you add a test so we can be sure this doesn't break in the future?

Thanks